rabbitmq/datadog_checks/rabbitmq/data/conf.yaml.example

```
@@ -38,6 +38,12 @@ instances:
     #
     # tag_families: false
 
+    ## @param collect_node_metrics - boolean - optional - default: true
+    ## Use the collect_node_metrics paramater if you'd like to collect
```

```suggestion
    ## Node metrics are collected by default. Disable the collect_node_metrics option to skip node metric collection.
```
Since this is collected by default, we can rephrase a bit?

---

Sure! That's clearer